encoding/protobuf/stream.go

```diff
@@ -60,13 +59,16 @@ func writeToStream(ctx context.Context, stream transport.Stream, message proto.M
 	return stream.SendMessage(
 		ctx,
 		&transport.StreamMessage{
-			Body: readCloser{Reader: bytes.NewReader(messageData), closer: cleanup},
+			Body: readCloser{
+				Reader: bytes.NewReader(messageData),
+				closer: cleanup,
+			},
 		},
 	)
 }
 
 type readCloser struct {
-	io.Reader
+	*bytes.Reader
```

Comment:
Why do you need to remove the interface here?

Reply:
`lenWrapper` assertion on fails on it as reader is enclosed in `io.Reader` interface